### PR TITLE
Make Juggernaut responsive

### PR DESCRIPTION
**Pull request by cpunk2140**

### CODE REVIEW

**Comment by cpunk2140** [comment]
@johncantrell97 Reopened the PR here. I also made the advanced node list used when opening channels horizontally scrollable. Might be a little confusing on Mac as it seems to not show the scrollbars by default? The "Open" is not initially visible if the screen is narrow. Not sure how much of a big deal it is, I could alternatively try to force it to show the scrollbar or display the action as the first column? Take a look and let me know how you feel about it overall.

**Comment by johncantrell97** [comment]
cool, I'll take a look now

**Comment by johncantrell97** [comment]
I see the scrollbars by default.  It is a little weird that open is not initially visible.  It's not a huge deal I guess but one solution I know a lot of websites do in this case is to "fix" a column so it's always visible and have the rest of them scroll.

I think for now let's just leave it.    

Everything else looks pretty good.  Still not in love with the launch wallet screen changes but I think it's fine.

Do you think 800px is the right breakpoint? On some screens it feels a little 'too soon' but I don't really feel too strongly.  Maybe 640px is better?

File: renderer/app.global.scss [hunk 1]
```diff
@@ -81,7 +87,6 @@ code {
 @import './css/common.scss';
 @import './css/chat.scss';
 @import './css/channels.scss';
-@import './css/connect.scss';
```
Comment (johncantrell97) [inline renderer/app.global.scss:84]: why'd you remove this?

**Pull request merged**